client/components/cocktail-list.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class Cocktails extends React.Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      list: props.cocktails
+    };
+  }
+
+  _addCocktails () {
+    return this.props.cocktails.map(function (cocktail) {
+      return <li>{cocktail.name}</li>;
+    });
+  }
+
+  render () {
+    return (
+      <ul>
+        {this._addCocktails()}
+      </ul>
+    );
+  }
+
+}
+
+Cocktails.propTypes = {
+  cocktails: React.PropTypes.arrayOf(React.PropTypes.shape({
+    name: React.PropTypes.string,
+    ingredients: React.PropTypes.array
```

Comment:
Oops, forgot, let's add the id too (now that we're gonna use it). Think it's a string, not a number?